src/js/common/schemaform/reducers/index.js

```diff
@@ -91,7 +91,22 @@ export default function createSchemaFormReducer(formConfig) {
   return (state = initialState, action) => {
     switch (action.type) {
       case SET_DATA: {
-        const newState = _.set([action.page, 'data'], action.data, state);
+        // TODO: Write the documentation that says this is a thing (persistent schema
```

Comment:
Hmm...probably, but given your PR (https://github.com/department-of-veterans-affairs/vets-website/pull/5172), I'm thinking this one should be changed to address only the schemaform reducer modification. I don't have any ideas for that test off-hand, though. When it becomes more pertinent, we can loop back and touch on it again.